bin/build-plugin-zip.sh

```diff
@@ -86,6 +86,7 @@ build_files=$(
 	build/edit-widgets/blocks/*/block.json \
 	build/widgets/blocks/*.php \
 	build/widgets/blocks/*/block.json \
+	build/style-engine/class-wp-style-engine-gutenberg.php \
```

Comment:
In case there is more files in the future :)
```suggestion
	build/style-engine/*.php \
```